bundled/tool/server.py

```diff
@@ -11,6 +11,8 @@
 import traceback
 from typing import Any, Callable, Dict, List, Optional, Sequence, Union
 
+from pygls import lsp, protocol, server, uris, workspace
```

Comment:
This needs to be moved back to line 39. This will break code otherwise. See the comment on line 37